components/connector-service/README.md

```diff
@@ -27,7 +27,8 @@ The Connector Service has the following parameters:
 - **tokenLength** - Length of registration tokens. The default value is `64`.
 - **appTokenExpirationMinutes** - Time after which tokens for Applications expire and are no longer valid. The default value is `5` minutes.
 - **runtimeTokenExpirationMinutes** - Time after which tokens for Runtimes expire and are no longer valid. The default value is `10` minutes.
-- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `nginx-auth-ca`.
+- **caSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains the certificate and key used for signing client certificates. The default value is `kyma-integration/nginx-auth-ca`.
+- **rootCACertificateSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains root CA (Certificate Authority) Certificate in case certificates are singed by intermediate CA.
```

Comment:
```suggestion
- **rootCACertificateSecretName** - Namespace and the name of the Secret which stores the root CA (Certificate Authority) Certificate in case the certificates are signed by the intermediate CA. Requires the `Namespace/secret name` format. 
```